### ISSUE TITLE
Use as an alternative to Magic Home Inventory?

### ISSUE THREAD

**Issue by jose1711** [jedie/PyInventory#100]
I am currently using Magic Home Inventory (https://play.google.com/store/apps/details?id=net.twisterrob.inventory&gl=US) on Android smartphone to keep track of items and locations at my apartment but the application does not integrate a shared access/multiple users approach so I am looking for something else.

One thing MHI is strong at is that you can have an item encapsulated in another item at will (https://www.twisterrob.net/project/inventory/#model). There is no concept of locations and this design works pretty well for my use case. As a consequence you can have things like `CR2032 => Rear bicycle light => Bicycle => Garage`.

I was trying to achieve the same with PyInventory but failed. Is there any chance to modify it in a way that would be very close to this design or I should look elsewhere?

Thank you.

**Comment by jedie** [comment]
This is possible: a item can have a parent item.
So you can create these items: CR2032 => Rear bicycle light => Bicycle.

And a location Garage.
Then "put" the Bicycle into the Garage by set the location on the Bicycle item to the Garage location 😉

**Comment by jose1711** [comment]
Thank you @jedie I gave it a shot but PyInventory does not give me the option to set a second level item as a parent. It only shows top-level item in the parent drop-down, that means I am able to do:
```
Rear bicycle light => Bicycle
CR2032 => Bicycle
```
Am I doing something wrong?

**Comment by jedie** [comment]
Yes, every item can have one one parent. But you should made a chain!
Maybe this is a other concept than magic home.

The CR2032 battery has as parent the light and this has as parent the Bicycle and this has no parent, but the location garage.

**Comment by jose1711** [comment]
Exactly, I am trying to make a chain. Will try to add some pictures. This is the current situation:

![obrázok](https://user-images.githubusercontent.com/1406222/179371347-6f0519e5-03a7-43ee-9198-e733ab312039.png)

Now I want battery to become a "child" of Rear light. So I open it:
![obrázok](https://user-images.githubusercontent.com/1406222/179371391-626ab18b-89f9-4808-85da-af356cc4bf02.png)

Now I expand parent dropdown:
![obrázok](https://user-images.githubusercontent.com/1406222/179371402-67503d48-9127-408b-8f96-5f9260737aa8.png)

And here is the problem - bicycle light cannot be selected (its entry is missing).

**Comment by jedie** [comment]
I'm work on a solution, looks like:

![grafik](https://user-images.githubusercontent.com/71315/180301035-85afb372-d238-4c00-8df1-e603d7a4089e.png)

![grafik](https://user-images.githubusercontent.com/71315/180301129-06f82130-e89c-4628-a6a3-a25c2b897255.png)

here: https://github.com/jedie/PyInventory/pull/106
